borgcron.sh

```diff
@@ -250,12 +250,13 @@ done
 # backup-type are touched.
 # ($PRUNE_PARAMS intentionally not quoted)
 
-if [ "$PRUNE_PARAMS" ] && [ "$PRUNE_PREFIX" != "null" ] && [ "$EXITCODE" -lt 2 ]; then
+if [ "$PRUNE_PARAMS" ] && [ "$PRUNE_PREFIX" != "null" ] && [ "$EXITCODE" -lt 1 ]; then
```

Comment:
Wait wait, why not prune if the exit code is "1"? What is the reason behind it?

If the backup fails, because some files may be missing/inaccessible/permission problems/etc. pruning should still be triggered. Otherwise users, who always have some of these "minor" problems, may never get old backups pruned.